src/ml_tooling/baseclass.py

```diff
@@ -408,7 +409,10 @@ def score_estimator(
         logger.info("Scoring estimator...")
 
         if not data.has_validation_set:
-            raise DatasetError("Must run create_train_test first!")
+            data.create_train_test(
+                stratify=True if self.is_classifier else False,
```

Comment:
`self.is_classifier` returns True or False, no need for the `if`

Reply:
The change has been implemented in the newest commit